toolkits/global/packages/global-article/scss/50-components/article-enhanced.scss

```diff
@@ -386,7 +386,7 @@
 	padding: 0;
 }
 
-@include media-query(map-get($article--breakpoints, 'lrg') map-get($article--breakpoints, 'med'), 'range') {
+@include media-query('lg' 'md', 'range') {
```

Comment:
Are these not backwards? From md - lg, rather than lg - md. No idea what difference it makes, if any.

Reply:
This appears to be correct, at least not changed from the original.

```@media only screen and (min-width: 1024px) and (max-width:767px) {
    .c-article-section__illustration-center,.c-article-section__illustration-right {
        margin:0 auto 24px;
        float: none;
        max-width: 100%
    }
}```